samples/BasicPubSub/src/main/java/pubsub/PubSub.java

```diff
@@ -251,19 +251,19 @@ public void onConnectionResumed(boolean sessionPresent) {
                 .withCleanSession(true)
                 .withProtocolOperationTimeoutMs(60000);
 
+            HttpProxyOptions proxyOptions = null;
```

Comment:
Couldn't this just be in the `if` scope?

Reply:
Yes